src/Raven.Server/Utils/Monitoring/DatabaseMetrics.cs

```diff
@@ -0,0 +1,156 @@
+                [nameof(DatabaseId)] = DatabaseId,
+                [nameof(UptimeInSec)] = UptimeInSec,
+                [nameof(TimeSinceLastBackupInSec)] = TimeSinceLastBackupInSec,
+                [nameof(Counts)] = Counts.ToJson(),
+                [nameof(Statistics)] = Statistics.ToJson(),
+                [nameof(Indexes)] = Indexes.ToJson(),
+                [nameof(Storage)] = Storage.ToJson()
+            };
+        }
+    }
+
+    public class DatabaseStatistics
+    {
+        public int DocPutsPerSecond { get; set; }
+        public int MapIndexIndexesPerSecond { get; set; }
+        public int MapReduceIndexMappedPerSecond { get; set; }
+        public int MapReduceIndexReducedPerSecond { get; set; }
+        public int RequestsPerSecond { get; set; }
+        public int RequestsCount { get; set; }
+        public int RequestAverageDuration { get; set; }
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(DocPutsPerSecond)] = DocPutsPerSecond,
+                [nameof(MapIndexIndexesPerSecond)] = MapIndexIndexesPerSecond,
+                [nameof(MapReduceIndexMappedPerSecond)] = MapReduceIndexMappedPerSecond,
+                [nameof(MapReduceIndexReducedPerSecond)] = MapReduceIndexReducedPerSecond,
+                [nameof(RequestsPerSecond)] = RequestsPerSecond,
+                [nameof(RequestsCount)] = RequestsCount,
+                [nameof(RequestAverageDuration)] = RequestAverageDuration
+            };
+        }
+    }
+
+    public class DatabaseCounts
+    {
+        public long Documents { get; set; }
+        public long Revisions { get; set; }
+        public long Attachments { get; set; }
+        public long UniqueAttachments { get; set; }
+        public long Alerts { get; set; }
+        public int Rehabs { get; set; }
+        public long PerformanceHints { get; set; }
+        public int ReplicationFactor { get; set; }
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(Documents)] = Documents,
+                [nameof(Revisions)] = Revisions,
+                [nameof(Attachments)] = Attachments,
+                [nameof(UniqueAttachments)] = UniqueAttachments,
+                [nameof(Alerts)] = Alerts,
+                [nameof(Rehabs)] = Rehabs,
+                [nameof(PerformanceHints)] = PerformanceHints,
+                [nameof(ReplicationFactor)] = ReplicationFactor
+            };
+        }
+    }
+    
+    public class DatabaseIndexesMetrics
+    {
+        public long Count { get; set; }
+        public int StaleCount { get; set; }
+        public long ErrorsCount { get; set; }
+        public int StaticCount { get; set; }
+        public int AutoCount { get; set; }
+        public int IdleCount { get; set; }
+        public int DisabledCount { get; set; }
+        public int ErrorCount { get; set; }
```

Comment:
Errored maybe?

Reply:
yes - this is better name - so errored mean amout of indexes with error state, and errors count is sum of indexing errors.

Reply:
I will describe each prop in telegaf input plugin readme file along with metric name.